marimo/_convert/converters.py

```diff
@@ -60,13 +61,39 @@ def from_non_marimo_python_script(
 
         Args:
             source: Unknown Python script source code string
+            aggressive: If True, will attempt to convert aggressively,
+                        turning even invalid text into a notebook.
         """
         from marimo._convert.non_marimo_python_script import (
             convert_non_marimo_python_script_to_notebook_ir,
+            convert_non_marimo_script_to_notebook_ir,
+        )
+
+        if aggressive:
+            notebook_ir = convert_non_marimo_script_to_notebook_ir(source)
+        else:
+            notebook_ir = convert_non_marimo_python_script_to_notebook_ir(
+                source
+            )
+        return MarimoConvert.from_ir(notebook_ir)
+
+    @staticmethod
+    def from_text(
```

Comment:
`from_text` might be a confusing name

Reply:
Not a fan either, but not really a fan of the alternatives
- `from_block`
- `from_blob`
- `from_string`

LMK if you have an idea

Reply:
`from_string` or `from_plain_text`. the doc `Convert text from a script into` is confusing since it may not be a script